stack.yaml

```diff
@@ -1,4 +1,5 @@
 resolver: lts-7.14
+
```

Comment:
Not having a whitespace there seems wrong, even though it is indeed unrelated, but i'm nitpicking here.